hack/release.sh

```diff
@@ -14,18 +14,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Usage: REGISTRY=xxx ORG=yyy IMAGE=zzz ./hack/release.sh [--tag]
+
 set -ex
 
```

Comment:
@jcrossley3 Jim, I am thinking of whether we should write release.sh the same way as in https://github.com/knative/serving/blob/master/hack/release.sh.

Images will be published under gcr.io/knative-releases/ as released artifacts.
Nightly build is put under: gcr.io/knative-nightly/, if necessary.

As long as we do source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/release.sh. These params have been used automatically.  The only thing we need to implement is a function called build_release.